src/nprintml/automl.py

```diff
@@ -0,0 +1,143 @@
+import pathlib
+import matplotlib.pyplot as plt
+
+import numpy as np
+import pandas as pd
+import autogluon as ag
+from autogluon import TabularPrediction as task
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelBinarizer
+from sklearn.metrics import roc_curve, auc, precision_recall_curve, \
+        average_precision_score, confusion_matrix, ConfusionMatrixDisplay
+
+import seaborn as sns
+sns.set_style("ticks")
+sns.set_context(rc={"lines.linewidth": 1.75})
```

Comment:
I'm of the opinion that we set defaults and provide the most basic use case graphs with reasonable defaults (i.e. we've used these defaults in papers) and if user's want more intense graphs we leave the models around for them to use.